tests/scanners/test.binary.js

```diff
@@ -32,4 +34,7 @@ describe('Binary', function() {
     assert.equal(BinaryScanner.fileStreamType, 'chunk');
   });
 
+  it('should report a proper scanner name', () => {
+    assert.equal(BinaryScanner.scannerName, 'binary');
```

Comment:
Should these be constants somewhere? They seem a bit like magic constants if we're using them elsewhere, but if it's just for reporting I guess that's okay. But if they're in more than one file we should probably make them constants.